tests/fixtures/aiohttp/openapi_multipart.yaml

```diff
@@ -6,11 +6,11 @@ info:
   title: "{{title}}"
   version: "1.0"
 paths:
-  "/mixed_single_file":
```

Comment:
Here, I just moved the bottom two endpoints to the top because that's the same order they are in in the test handler and test requester files